client/src/actions/wind-sensor.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable unicorn/consistent-function-scoping */
-import axios from 'axios';
 import moment from 'moment';
+import axios from '../utils/fetch-with-auth';
```

Comment:
It confuses, please rename imports into `client` or `apiClient` everywhere

Reply:
you mean instead of `axios`?

Reply:
yes, everywhere where `axios` is not an actual axios but your wrapper